CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+We also release the software as open source software so others can review it.
+
+Since early detection and impact reduction can never be perfect, we also try to detect and repair problems during deployment as quickly as possible.
+
+## <span id="how_to_report_vulnerabilities">Vulnerability reporting (security issues)</a>
+
+If you find a significant vulnerability, or evidence of one, please send an email to the security contacts that you have such information, and we'll tell you the next steps.
+For now, the security contacts are:
+IUDX admin <[admin@iudx.org.in](mailto:support@iudx.org.in)> and 
+IUDX Support <[support@iudx.org.in](mailto:support@iudx.org.in)>.
+
+Please use an email system (like Gmail) that supports hop-to-hop encryption using STARTTLS when reporting vulnerabilities. Examples of such systems include Gmail, Outlook.com, and runbox.com. 
+See [STARTTLS Everywhere](https://starttls-everywhere.org/) if you wish to learn more about efforts to encourage the use of STARTTLS.
+Your email client should use encryption to communicate with your email system (i.e., if you use a web-based email client then use HTTPS, and if you use email client software then configure it to use encryption). Hop-to-hop encryption isn't as strong as end-to-end encryption,
+but we've decided that it's strong enough for this purpose and it's much easier to get everyone to use it.
+
+We will gladly give credit to anyone who reports a vulnerability
+so that we can fix it. If you want to remain anonymous or pseudonymous instead,
+please let us know that; we will gladly respect your wishes.
+
+## Documentation changes
+
+Most of the documentation is in "markdown" format. All markdown files use the .md filename extension.
+
+## Code changes
+
+To make changes to the "aaa-server" ,you may find the following helpful; [README.md](https://github.com/datakaveri/iudx-aaa-server/blob/main/README.md)
+installation and implementation information.
+
+The code should strive to be DRY (don't repeat yourself), clear, and obviously correct.
+Some technical debt is inevitable, just don't bankrupt us with it. Improved refactorizations are welcome.
+
+### Automated tests
+
+When adding or changing functionality, please include new tests for them as
+part of your contribution.
+
+We require the Java code to have at least 90% statement coverage;
+please ensure your contributions do not lower the coverage below that minimum.
+The Java code uses Junit as the test framework. Additional tests are very welcome.
+
+We encourage tests to be created first, run to ensure they fail, and then add code to implement the test (aka test driven development).
+
+
+### Other tools
+
+Here are some other tools we sometimes use for checking quality or security.
+
+* OWASP ZAP web application security scanner.
+  You are encouraged to use this and other web application scanners to find and
+  fix problems.
+
```

Comment:
Add this section
### Testing during continuous integration

Note that we also use
[Jenkins](https://jenkins.iudx.io/)
for continuous integration tools to check changes
after they are checked into GitHub; if they find problems, please fix them.
These run essentially the Junit tests, JaCoCo code coverage tests, security tests using OWASP ZAP and integration tests using postman and newman